source/ingress.go

```diff
@@ -17,24 +17,32 @@ limitations under the License.
 package source
 
 import (
+	"html/template"
```

Comment:
it would be nice to have the imports structured, 

std libs
vendor packages
local packages